dactyl-config.yml

```diff
@@ -2783,6 +2863,13 @@ pages:
         # Japanese: XRP Ledgerに対する公式なクライアントライブラリです。JavaScriptのみで使用できます。
```

Comment:
You can delete this line since it's replaced by the new translated blurb on line 2870